### PR TITLE
Release 1.4.0 into trunk

### PR DESCRIPTION
**Pull request by AliSoftware**
New version 1.4.0. Be sure to create a GitHub Release and tag once this PR gets merged.

### CODE REVIEW

**Comment by mokagio** [comment]
I left a comment regarding the failing GitHub Actions build [here](https://github.com/wordpress-mobile/release-toolkit/pull/292#issuecomment-891758279).

**Comment by mokagio** [comment]
Opened #293 to address the CI issue.

**Review by mokagio — APPROVED** [review]
LGTM. 

Once #293 gets into the release branch, we should be able to merge and ship.

File: CHANGELOG.md [hunk 1]
```diff
@@ -28,6 +39,9 @@ _None_
 * Fix the way versioning is handled for alphas – i.e. `version.properties` is indexed by flavor name, defaulting to `zalpha` for alphas. [#283]
 * Fixed an issue in `check_translation_progress` where a wrong evaluation of the progress is possible when there are Waiting string in GlotPress.
 
+### Internal Changes
+
+* Opt-out from installing platform-specific gems with Bundler [#293]
```
Comment (AliSoftware) [inline CHANGELOG.md:44]: This is in the wrong place, should be in 1.4.0 not 1.3.1

**Pull request merged**